packages/twenty-front/src/modules/settings/integrations/components/SettingsIntegrationSyncStatusToggle.tsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+
+import { Toggle } from '@/ui/input/components/Toggle';
+import { RemoteTableStatus } from '~/generated-metadata/graphql';
+
+export const SettingsIntegrationSyncStatusToggle = ({
+  item: table,
```

Comment:
This component seems to be specific to the RemoteTable settings. The `item` prop can be named `table` directly, and the name of the component could include `RemoteTable` for clarity (`SettingsIntegrationsRemoteTableSyncStatusToggle`).